recipes/libtheora/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: libtheora
+  version: 1.1.1
+
+source:
+  fn: libtheora-1.1.1.tar.bz2
+  url: http://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.bz2
+  md5: 292ab65cedd5021d6b7ddd117e07cd8e
+
+build:
+    number: 0
+    skip: True  # [not linux]
+
+requirements:
+  build:
+    - pkg-config
+    - toolchain
+    - libogg
+    - libvorbis
+    - libpng <1.6
+    - zlib
+
+  run:
+    - libogg
+    - libvorbis
+    - libpng <1.6
+    - zlib
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libtheora.so
+    - test -f ${PREFIX}/lib/libtheoradec.so
+    - test -f ${PREFIX}/lib/libtheoraenc.so
+    - test -f ${PREFIX}/lib/libtheora.a
+    - test -f ${PREFIX}/lib/libtheoradec.a
+    - test -f ${PREFIX}/lib/libtheoraenc.a
+
+about:
+  home: https://theora.org
+  license: BSD
```

Comment:
mh ...
Theora (and all associated technologies released by the Xiph.org Foundation) is released to the public via a BSD-style license. It is completely free for commercial or noncommercial use. That means that commercial developers may independently write Theora software which is compatible with the specification for no charge and without restrictions of any kind.